src/modules/react-reconciler/Atlas.tsx

```diff
@@ -483,7 +488,11 @@ export const Atlas: React.FC<
           ref={refs.canvas as any}
         />
       )}
-      <Container className="atlas-overlay" style={{ ...(overlayStyle || {}) }} ref={refs.overlay as any}>
+      
+      <Container className={['atlas-overlay', isInteractive ? 'atlas-overlay--interactive' : null]
```

Comment:
Elsewhere Stephen uses `''` instead of `null` -- I wonder if we should be consistent